.github/workflows/deploy_production.yml

```diff
@@ -51,7 +51,7 @@ jobs:
     name: Production
     needs: [guard]
     if: ${{ needs.guard.outputs.should_deploy == 'true' }}
-    uses: primer/.github/.github/workflows/deploy.yml@0cec9b9914f358846163f2428663b58da41028c9
+    uses: primer/.github/.github/workflows/deploy.yml@bb0b18fb589038a93a49a24624be0559c01ed0b9
```

Comment:
How did you find this hash?

Reply:
@pksjce honestly just looking at the latest commit in the repo 😂 

It'd be great if we could tag our stuff in `primer/.github` so that we don't have to use hashes and so that dependabot can pick up on updates